spec/decanter/parser/boolean_parser_spec.rb

```diff
@@ -61,5 +75,33 @@
           .to raise_error(Decanter::ParseError)
       end
     end
+
+    context 'returns true with options for' do
+      trues_with_options.each do |cond|
+        it "#{cond[0]}: #{cond[1]}, option: {#{cond[2]}: #{cond[3]}}" do
+          expect(parser.parse(name, cond[1], true_value: cond[3])).to match({name => true})
```

Comment:
Out of curiosity, is the older hashrocket syntax required?

Reply:
no it is not, but it seems that this spec file and some others use it. Might be worth updating all tests in a separate PR?

Reply:
Makes sense to me! Happy to consider that out of scope here